cmd/minishift/cmd/root.go

```diff
@@ -117,14 +128,49 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
-		ensureConfigFileExists(constants.ConfigFile)
+		// Check if old instance state config file exists and move it to new one.
+		// Create MACHINE_NAME-state.json (machine state config file)
```

Comment:
suggestion: `If older instance state config file exists and copy it to then rename the file else create a the new file`.